rails_docker.rb

```diff
@@ -29,7 +29,7 @@ def source_paths
 # Gemfile
 remove_file 'Gemfile'
 copy_file 'rails_docker/Gemfile.txt', 'Gemfile'
-copy_file 'rails_docker/.rvmrc', '.rvmrc'
+copy_file 'rails_docker/.ruby-version', '.ruby-version'
```

Comment:
Is this compatible with an version package manager?

Reply:
Yes @olivierobert This is warning on my terminal when I use `.rvmrc`

![screen shot 2017-11-20 at 11 17 30 am](https://user-images.githubusercontent.com/11751745/33002231-6e4644ba-cde5-11e7-855b-4977d6dd6bbb.png)

Reply:
@anduonghien @olivierobert In the [DOCS](https://rvm.io/workflow/projects#project-file-rvmrc),

> `.rvmrc` has one major flaw - it requires trusting to prevent execution of unauthorized code, which makes it hard to use and complicates deployment to production. **In cases where scripting is not required, use of other project files is encouraged.**

I think `.ruby-version` file is a good-fit for us.

But I suggest using the shell command,

`rvm --ruby-version use [RUBY_VERSION]@[GEMSET]`

This way we can generate the ruby-version file and the gemset file at once.